backends/p4tools/modules/testgen/README.md

```diff
@@ -2,8 +2,6 @@
 
```

Comment:
The header is missing here.

Reply:
The first line is `# P4Testgen`.  I am not sure which header you are referring to.

Reply:
Haven't had my coffee yet, never mind :)